draft-ietf-quic-spin-exp.md

```diff
@@ -224,11 +214,12 @@ small amount of periodic application traffic, where that period is longer than
 the RTT, measuring the spin bit provides information about the application
 period, not the network RTT.
 
-Simple heuristics based on the observed data rate per flow or changes in the
-RTT series can be used to reject bad RTT samples due to application or flow
-control limitation; for example, QoF {{TMA-QOF}} rejects component RTTs
-significantly higher than RTTs over the history of the flow. These heuristics
-may use the handshake RTT as an initial RTT estimate for a given flow.
+Simple heuristics based on the observed data rate per flow or changes in the RTT
+series can be used to reject bad RTT samples due to lost or reordered edges in
+the spin signal, as well as application or flow control limitation; for example,
+QoF {{TMA-QOF}} rejects component RTTs significantly higher than RTTs over the
+history of the flow. These heuristics may use the handshake RTT as an initial
+RTT estimate for a given flow.
```

Comment:
I would be careful here because you actually want to detect suddent changes. The question is rather is it only one outlier or did actually something happen.

Reply:
yep, but this is a common problem with any passive RTT scheme (or indeed, RTT estimation at an endpoint): the base data is noisy, so it takes a few samples to adjust to a new real value.